docs/source/user-guide/general/deploying_on_ec2.rst

```diff
@@ -0,0 +1,11 @@
+====================
+Deploying on AWS EC2
+====================
+
+Minimum Requirements
+--------------------
+A linux distro i.e Amazon Linux and we recommend atleast 4 cores and 8GB of ram.
+
+Network Requirements
+--------------------
+Have 443 and 8433 ports open.
```

Comment:
Change to:
Ports 443 and 8433 open to outbound communication. 

@JKL98ISR we should recommend a full solution, which means the EC2 instance is not opened to the world, but connects to the world via a load balancer. so 443 and 8433 are opened only to the LB.